src/main/java/duke/DukeList.java

```diff
@@ -0,0 +1,94 @@
+package duke;
+
+import task.*;
+import exception.*;
+
+import java.time.LocalDate;
+import java.io.Serializable;
+import java.util.ArrayList;
+
+public class DukeList implements Serializable {
+    private ArrayList<Task> list = new ArrayList<>();
+
+    public DukeList() {}
+
+    private String pluralTask (int size) {
+        if (size == 1) {
+            return " task.";
+        } else {
+            return " tasks.";
+        }
+    }
+
+    public void add(String type, String s) {
+        Task task = new Task();
+        if (type.equals("todo")) {
+            task = new Todo(s);
+
+        } else if (type.equals("deadline")) {
+            String[] arr = s.split(" /by ", 2);
+            LocalDate localDate = LocalDate.parse(arr[1]);
+            task = new Deadline(arr[0], localDate);
+
+        } else {
+            String[] arr = s.split(" /from | /to", 3);
+            task = new Event(arr[0], arr[1], arr[2]);
+        }
+        System.out.println("Sure, Imma add that real quick");
+        System.out.println(task);
+        list.add(task);
+        System.out.println("Now you've got " + list.size() + pluralTask(list.size()));
+        System.out.println(new TextBorder(""));
+    }
```

Comment:
Maybe you can move all the prints into the Ui class? After all you have a Ui class and the Ui class should deal with interactions with the user